episodes/chap8_own_project.md

```diff
@@ -47,9 +48,25 @@ algorithm. Now, you can add privacy filters to your algorithm.
   exceptions](https://docs.vantage6.ai/en/main/function-docs/_autosummary/vantage6.algorithm.tools.exceptions.html)
   to return an error message.
 
-Hint: the [contingency table algorithm](https://github.com/vantage6/v6-crosstab-py)
+:::::::::::::::: hint
```

Comment:
It's not a new addition but I would put "- What else could you do to protect the privacy of the data?" as the last question, maybe also set it apart from the rest because it's an actual question while the rest are instructions.